File: python/1038.py
'''
    Autor: Guiventura6;
    Data: 06/03/2022;
    Problema: Snack;
    Resume:Program that reads a cod and the amount of a product and calculates the value
    to pay
'''
def calculate_payment(cod_product, units):
    products = {
        (1, 'Cachorro Quente', 4.00),
        (2, 'X-Salada', 4.50),
        (3, 'X-Bacon', 5.00),
        (4, 'Torrada simples', 2.00),
        (5, 'Refrigerante', 1.50)
    }
    payment = 0.0
    for product in products:
        cod, name_product, price = product
        if cod == cod_product:
            payment = price * units
            break
    return payment

if __name__ == '__main__':

    code, units = map(int, input().split())

    print('Total: R$ {:.2f}'.format(calculate_payment(code, units)))

File: c/1038.c
/*  
    @autor: Guiventura6;
    @data: 21/01/2022;
    @Problema: Snack;
*/
#include <stdio.h>

int main(){

    int codProduct, amount;
    double total;

    scanf("%d%d", &codProduct, &amount);

    switch (codProduct) {
    case 1:
        //Cachorro Quente - Price: 4.00
        total = amount * 4.00;
        break;
    case 2: 
        //X-Salada - Price: 4.50
        total = amount * 4.50;
        break;
    case 3: 
        //X-Bacon - Price: 5.00
        total = amount * 5.00;
        break;
    case 4: 
        //Torrada Simples - Price: 2.00
        total = amount * 2.00;
        break;
    case 5: 
        //Refrigerante - Price: 1.50
        total = amount * 1.50;
        break;    
    default:
        break;
    }
    
    printf("Total: R$ %.2lf\n", total);

    return 0;
}